### PR TITLE
Update SABnzbd 3.5.1

### PR DESCRIPTION
**Pull request by Safihre**
## Description

Update SABnzbd 3.5.1.

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Update-Policy#tests-checks)
- [x] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [ ] Bug fix
- [ ] New Package
- [x] Package update
- [ ] Includes small framework changes
- [ ] This change requires a documentation update (e.g. Wiki)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,5 +11,6 @@ toolchain/*/work*
 toolchain/*/*_done
 toolkit/*/work*
 local.mk
+.idea/
 *.DS_Store
```
Comment (th0ma7) [inline .gitignore:15]: Why `.idea` ?

**Review by th0ma7 — COMMENTED** [review]
Other than the `.idea` which I wonder why, everything else looks good otherwise.

File: .gitignore (hunk 1 above)
Comment (Safihre) [reply .gitignore:15]: It's the PyCharm project directory, will remove that,

**Review by th0ma7 — APPROVED** [review]

**Comment by th0ma7** [comment]
Pleasure!

**Pull request merged**